getting_started/intel_debug/introduction.rst

```diff
@@ -137,4 +137,36 @@ drivers and should work when changing the ``dsp_driver`` parameter.
 The selection of firmware, topology, and UCM files is based on platform
 capabilities, codec names, and DMI options. While the SOF team and the
 community try to cover all possible cases, errors will happen when the
-wrong file is selected at any of the three layers.
\ No newline at end of file
+wrong file is selected at any of the three layers.
+
+4. Chromebooks and SOF
+----------------------
+
+As stated above, Chromebook firmware is as of 2021 configured with the
+*community* key which means Chromebooks can run audio firmware signed by
+anyone. The entire filesystem is locked by default instead but there are
+a couple options to disable security for development purposes.
+
+Chromebooks can run Linux in several, non-mutually exclusive ways:
+
+- ChromeOS has direct hardware access but development cannot happen on
+  ChromeOS itself, it requires a separate workstation similar to how
+  most embedded development does. Setting up its ``cros_sdk`` requires
+  significant time and effort, see the `ChromeOS developer guide
```

Comment:
time yes, but it is "supposed" to be low effort... that being said, you do not need the cros_sdk to do strictly FW dev, if you want to modify the kernel then yes

Reply:
Good point about the kernel, thanks. I tried to avoid making a difference between SOF fw and the kernel (similar problem with Crouton) to keep this section short but now I think it's too much of a shortcut.